secator/ai/utils.py

```
@@ -1,31 +1,447 @@
+)
+
+
+def _sanitized_env() -> dict:
+	"""Return a copy of os.environ with sensitive variables removed.
+
+	Passed as the `env` run_opt to the AI shell `command` runner so an AI-run
+	`env`/`printenv` can't dump the LLM key + cloud creds into output that flows
+	back to the LLM and is persisted to Mongo.
+	"""
+	return {k: v for k, v in os.environ.items()
+			if not any(k.startswith(p) for p in SENSITIVE_ENV_PREFIXES)
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)